.changeset/changelog-config.js

```diff
@@ -0,0 +1,112 @@
+// @ts-check
```

Comment:
Copied this code from Hydrogen. It results in the following changelog format (see 1.47.0) -

This assumes you ran `yarn changeset` twice. For both minor + fix change